config_models/tests/test_config_models.py

```diff
@@ -196,17 +210,18 @@ class ExampleKeyedConfig(ConfigurationModel):
     """
     cache_timeout = 300
 
-    KEY_FIELDS = ('left', 'right')
+    KEY_FIELDS = ('left', 'right', 'user')
 
     left = models.CharField(max_length=30)
     right = models.CharField(max_length=30)
+    user = models.ForeignKey('auth.User')
```

Comment:
So this `KEY_FIELDS` change and `user` addition necessitates all the `user` changes below, correct?

Reply:
Yeah.